AirCasting/SessionsSynchronization/Controller/SessionSynchronizationController.swift

```diff
@@ -13,6 +13,7 @@ final class SessionSynchronizationController: SessionSynchronizer {
     private let downstream: SessionDownstream
     private let upstream: SessionUpstream
     private let store: SessionSynchronizationStore
+    private var settings: BaseURLProvider
```

Comment:
well it's not settings, it's `urlProvider` :) how about changing this name to be more precise?

Also - this should really not be shoved in here like that. This class should *only* be responsible for what it's meant to be doing - synchronization. It's really not a good idea to put side effects like that here. Try to change it so that we don't introduce this dependency and let's be a little bit more careful where we put logic in

Reply:
I know that this probably won't be a great idea but I made a public var for now - because I really can't think about something simple yet so efficient